react/features/invite/functions.js

```diff
@@ -568,8 +570,16 @@ export function getShareInfoText(
                     i18next.t('info.dialInConferenceID')} ${
                     conferenceID}#\n\n`;
             })
-            .catch(error =>
-                logger.error('Error fetching numbers or conferenceID', error))
+            .catch(error => {
+                logger.error('Error fetching numbers or conferenceID', error);
+                if (error?.status === StatusCode.PAYMENT_REQUIRED) {
+                    infoText += `${
+                        i18next.t('info.dialInNumber')} ${i18next.t('info.reachedLimit')} ${
+                        i18next.t('info.upgradeOptions')} ${UPGRADE_OPTIONS_TEXT}`;
+
+                    return infoText;
+                }
+            })
             .then(defaultDialInNumber => {
```

Comment:
since `.then` is chained after the `.catch`, in case .catch enters the `if (error?.status === StatusCode.PAYMENT_REQUIRED) {` - it will return infoText - so I assume in this case `defaultDialInNumber ` will actually have the value of the returned `infoText` . Is this what we want?

Reply:
yup, this `then` acts as a `finally`, which gets executed for both success and exceptions

Reply:
👍